website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -4,7 +4,22 @@ id: "core"
 description: "Learn about semantic versioning for dbt Core, and how long those versions are supported."
 ---
 
-dbt Core releases follow [semantic versioning](https://semver.org/) guidelines. For more on how we use semantic versions, see [How dbt Core uses semantic versioning](#how-dbt-core-uses-semantic-versioning).
+dbt Core releases follow [semantic versioning](https://semver.org/) guidelines. For more on how we use semantic versions, see [How dbt Core uses semantic versioning](#how-dbt-core-uses-semantic-versioning). 
+
+dbt Labs provides different support levels to different versions, which may include new features, bug fixes, or security patches:
+<!--
+- [For versions prior to v1.0](#version-support-prior-to-v1.0)
```

Comment:
i added this here bc i thought it would be a nice intro into the types of supported versions but commented out.